docs/src/makie.md

```diff
@@ -0,0 +1,79 @@
+# Makie.jl plots
+
+This page shows an example of plotting MCMCChains.jl with Makie.jl.
+The example is meant to provide an useful basis to build upon.
+Let's define some random chain and load the required packages:
+
+```@example makie
+using CairoMakie
+using DataFrames
+using MCMCChains
+
+chns = Chains(randn(300, 5, 3), [:A, :B, :C, :D, :E])
+```
+
+A basic way to visualize the chains is to show the drawn samples at each iteration.
+Colors depict different chains:
+
+```@example makie
+params = names(chns, :parameters)
+
+df = DataFrame(chns)
+n_chains = length(unique(df.chain))
+n_samples = nrow(df) / n_chains
+
+# Alternatively, use `CategoricalArrays.categorical`.
+df[!, :chain] = string.(df.chain)
+
+fig = Figure(; resolution=(1_000, 800))
+
+# Create and store separate axes for showing iterations.
+values_axs = [Axis(fig[i, 1]; ylabel=string(c)) for (i, c) in enumerate(params)]
+for (ax, col) in zip(values_axs, params)
+    for i in 1:n_chains
+        chain = string(i)
+        values = filter(:chain => ==(chain), df)[:, col]
```

Comment:
One could use TableOperations.jl.

Reply:
I tried and couldn't get it to work.

One could also grab the right matrix row from the AxisArray :sweat_smile: This is what is implemented now
```julia
julia> get(chns, :A)[:A]
2-dimensional AxisArray{Float64,2,...} with axes:
    :iter, 1:1:300
    :chain, 1:3
And data, a 300×3 Matrix{Float64}:
  2.26218      0.214943    1.01596
 -0.428555     0.603559   -0.47663
 -0.626586    -0.819862    1.59953
[...]
```

Reply:
Just use regular indexing, ie. `chn[:A]`? BTW one should never have to work with the underlying axis array directly,ie never use `chn.values`.